subjects/DPxAI/play-with-variables/README.md

```diff
@@ -0,0 +1,93 @@
+## Play with variables
+
+> Mindful AI mode
+
+### Context
+
+Remember that if things get a little hectic at times, take the time to get closer to your peers so that you can think, share and move forward together.
+
+> Keep Going!
+
+### AI-Powered Learning Techniques
+
+**Clarification Technique:**
+
+This type of prompt encourages the AI to explain a concept in detail, helping you gain a deeper understanding.
+
+> Find the examples across the subject ;)
+
+## Concepts
+
+### Escape characters
+
+**Quote delimiters** can be one of the tricky things to deal with.
+
+Since they are used for delimiting text, they need a trick to include them in
+our text.
+
+For example, we want a `'` _(single quote)_ in our text, but use them as
+delimiters:
+
+```js
+console.log('I keep trying , I can't give up! ')
+// too bad a single quote, ruined the quote, get it ?
+```
+
+The `\` _(backslash)_ is used for that:
+
+Every time there is an _extra special_ character into your string, putting a `\`
+in front of it will **escape** it and doing so will let JS understand you meant
+the **literal** following character and not the delimiter, _or whatever else
+the character normally means for a string_
+
+```js
+console.log("I keep trying , I can't give up! ");
+
+// Output: I keep trying, I can't give up!
+```
+
+#### **`Prompt Example`**:
```

Comment:
This displays a bit odd:
![image](https://github.com/user-attachments/assets/1ec30d59-52c6-4ebc-b6a1-28585bf40959)
should we change?

Reply:
Fixed.